src/odin/enhancedtrippath.cc

```diff
@@ -27,6 +27,7 @@ constexpr int kIsStraightestBuffer = 10;                   // Buffer between str
 constexpr uint32_t kBackwardTurnDegreeLowerBound = 124;
 constexpr uint32_t kBackwardTurnDegreeUpperBound = 236;
 
+#ifdef LOGGING_LEVEL_TRACE
```

Comment:
we have lots of stuff like this in `odin` as well and i hate it. logging directives seeping out into header files and everything which is terrible. the bulk of this stuff is used for debug printing when analysing routes

Reply:
For the moment less intrusive to mark when to compile the loggings with the directive.
I will keep in mind when I see the other directories to think about that strategy